src/Lab01/End/Gateway/Gateway/Gateway.csproj

```diff
@@ -1,16 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk.Web">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.2</TargetFramework>
-    <AspNetCoreHostingModel>InProcess</AspNetCoreHostingModel>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
+    <UserSecretsId>99b16f1a-e7df-4cf5-8bf4-2f7fb4d38a90</UserSecretsId>
```

Comment:
To avoid complexity for the lab, I wonder if we should back this out and stick with port 80?